prepare_vscode.sh

```diff
@@ -54,53 +66,86 @@ else
-linkProtectionTrustedDomains='setpath(["linkProtectionTrustedDomains"]; ["https://open-vsx.org"])'
-nameShort='setpath(["nameShort"]; "VSCodium")'
-nameLong='setpath(["nameLong"]; "VSCodium")'
-linuxIconName='setpath(["linuxIconName"]; "vscodium")'
-applicationName='setpath(["applicationName"]; "codium")'
-win32MutexName='setpath(["win32MutexName"]; "vscodium")'
-win32DirName='setpath(["win32DirName"]; "VSCodium")'
-win32NameVersion='setpath(["win32NameVersion"]; "VSCodium")'
-win32RegValueName='setpath(["win32RegValueName"]; "VSCodium")'
-win32AppUserModelId='setpath(["win32AppUserModelId"]; "Microsoft.VSCodium")'
-win32ShellNameShort='setpath(["win32ShellNameShort"]; "VSCodium")'
-win32x64UserAppId='setpath (["win32x64UserAppId"]; "{{2E1F05D1-C245-4562-81EE-28188DB6FD17}")'
-urlProtocol='setpath(["urlProtocol"]; "vscodium")'
-serverApplicationName='setpath(["serverApplicationName"]; "codium-server")'
-serverDataFolderName='setpath(["serverDataFolderName"]; ".vscodium-server")'
-reportIssueUrl='setpath(["reportIssueUrl"]; "https://github.com/VSCodium/vscodium/issues/new")'
-licenseUrl='setpath(["licenseUrl"]; "https://github.com/VSCodium/vscodium/blob/master/LICENSE")'
-
-product_json_changes="${checksumFailMoreInfoUrl} | ${tipsAndTricksUrl} | ${twitterUrl} | ${requestFeatureUrl} | ${documentationUrl} | ${introductoryVideosUrl} | ${updateUrl} | ${releaseNotesUrl} | ${keyboardShortcutsUrlMac} | ${keyboardShortcutsUrlLinux} | ${keyboardShortcutsUrlWin} | ${quality} | ${extensionsGallery} | ${linkProtectionTrustedDomains} | ${nameShort} | ${nameLong} | ${linuxIconName} | ${applicationName} | ${win32MutexName} | ${win32DirName} | ${win32NameVersion} | ${win32RegValueName} | ${win32AppUserModelId} | ${win32ShellNameShort} | ${win32x64UserAppId} | ${urlProtocol} | ${serverApplicationName} | ${serverDataFolderName} | ${reportIssueUrl} | ${licenseUrl}"
-cat product.json.bak | jq "${product_json_changes}" > product.json.tmp
-
-jq -s '.[0] * .[1]' product.json.tmp ../product.json > product.json
-rm -f product.json.tmp
+setpath "product" "checksumFailMoreInfoUrl" "https://go.microsoft.com/fwlink/?LinkId=828886"
+setpath "product" "documentationUrl" "https://go.microsoft.com/fwlink/?LinkID=533484#vscode"
+setpath_json "product" "extensionsGallery" '{"serviceUrl": "https://open-vsx.org/vscode/gallery", "itemUrl": "https://open-vsx.org/vscode/item"}'
+setpath "product" "introductoryVideosUrl" "https://go.microsoft.com/fwlink/?linkid=832146"
+setpath "product" "keyboardShortcutsUrlLinux" "https://go.microsoft.com/fwlink/?linkid=832144"
+setpath "product" "keyboardShortcutsUrlMac" "https://go.microsoft.com/fwlink/?linkid=832143"
+setpath "product" "keyboardShortcutsUrlWin" "https://go.microsoft.com/fwlink/?linkid=832145"
+setpath "product" "licenseUrl" "https://github.com/VSCodium/vscodium/blob/master/LICENSE"
+setpath "product" "linkProtectionTrustedDomains" '["https://open-vsx.org"]'
+setpath "product" "linuxIconName" "vscodium"
```

Comment:
Shouldn't this be different for an insider build?
If we don't have one then just change blue to green?

Reply:
maybe for `linuxIconName`...

Reply:
I've update the property. Thx